src/datasources/memory.datasource.ts

```diff
@@ -0,0 +1,6 @@
+import { juggler, DataSourceConstructor } from '@loopback/repository';
```

Comment:
looks like `juggler` is not needed.

Reply:
You're right Janny, but if I remove it, I get the following compilation error:
```
[ts] Exported variable 'db' has or is using name 'juggler.DataSource' from external module "/Users/badmike/loopback/loopback-next-hello-world/node_modules/@loopback/repository/lib/loopback-datasource-juggler" but cannot be named.
```

Reply:
Yeah, I have encountered this problem myself, I am not aware of any better solution than to include `juggler` in imports, even if it's not directly referenced anywhere.